Dockerfile.ms

```diff
@@ -0,0 +1,62 @@
+#
+# ----- Go Builder Alpine Image ------
+#
+FROM golang:1.9-alpine AS builder
+
+RUN apk add --no-cache git bash || apk update && apk upgrade
```

Comment:
What is the purpose of `|| app update && app upgrade` on every line?

That is, if I’m not mistaken, parsed as (`apk add` || `apk upgrade`) && (`apk upgrade`), i.e. running “update” if “add” fails, and running “upgrade” always (and failing only if “upgrade” fails).